src/message.rs

```diff
@@ -172,8 +188,7 @@ pub struct Message {
     pub(crate) in_reply_to: Option<String>,
     pub(crate) server_folder: Option<String>,
     pub(crate) server_uid: u32,
-    // TODO: enum
-    pub(crate) is_dc_message: u32,
+    pub(crate) is_dc_message: MessengerMessage,
```

Comment:
I was resisting the temptation to bikeshed on the MessengerMessage name until I got here.  but now the variable is called is_dc_message I'd like to point out that MessengerMessage could also be called DcMessage.